tests/playwright/src/index.ts

```diff
@@ -19,6 +19,7 @@
 // export core modules
 export * from './globalSetup/global-setup';
 export * from './runner/podman-desktop-runner';
+export * from './runner/runner-options';
```

Comment:
I don't know if you want to export Runner under the name PodmanDesktopRunner for a couple of weeks the time to have changes in the consumer side